starter-core/src/main/java/io/micronaut/starter/feature/validator/HibernateValidator.java

```diff
@@ -13,17 +13,15 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package io.micronaut.starter.feature.other;
+package io.micronaut.starter.feature.validator;
```

Comment:
dont change the package

Reply:
resolved via https://github.com/micronaut-projects/micronaut-starter/pull/1717/commits/52687ebdd4cf8c507d807c2d7e4d74ff1124088d